### ISSUE TITLE
[sandboxie] Use sandboxie/sandboxie repo

### ISSUE THREAD

**Issue by Jackenmen** [jack1142/choco-auto#2]
sandboxie/sandboxie repo is maintained by one of original Sandboxie's core developers, so it should be closer to Sophos's Sandboxie therefore being a better choice for chocolatey package which used to install Sophos's Sandboxie. It's important to do this ASAP before the currently used fork is gonna stray away from original Sandboxie a lot.
It might be worth making a second chocolatey package for David's fork if he decides to continue on maintaining fork after becoming part of Sandboxie's org

**Comment by Jackenmen** [comment]
Update: I pushed an update to the sandboxie.install's update.ps1 to prevent more updates from sandboxie-plus repo. I'm still waiting for first release on https://github.com/sandboxie/sandboxie

**Comment by BlueDrink9** [comment]
I see releases of the original version of Sandboxie on the Plus github page :)

https://github.com/sandboxie-plus/Sandboxie/releases

**Comment by isaak654** [comment]
Update: [David's fork](https://github.com/sandboxie-plus/Sandboxie) is currently maintaining both Sandboxie & Sandboxie Plus with over 800 commits and his latest releases didn't receive any malware detection anymore (the latest one was a month ago, related to Google Safe Browsing). 

The original [Sandboxie repository](https://github.com/sandboxie/sandboxie) is still without releases, and [here](https://www.wilderssecurity.com/threads/sandboxie-technologies-sbie-open-source.428156/page-38#post-3001020) you can read the latest news about the maintainer.

I would appreciate if we can be unite on the same repository.

**Comment by Jackenmen** [comment]
I didn't update the status on this issue but I've been meaning to use Sandboxie-Plus's Sandboxie as well as make new package for Sandboxie Plus for very long time but just didn't have will to do it, especially that I don't use Sandboxie anymore. But I'll consider to find some time to do it in September

**Comment by Jackenmen** [comment]
I updated the `sandboxie`/`sandboxie.install` package to use Sandboxie Classic from David Xanatos and the new version has been pushed. The package is approved as a trusted package so it should pass moderation within the next 24-48 hours.

I have also made a new `sandboxie-plus`/`sandboxie-plus.install` package (no `sandboxie-plus.portable` for now but the application supports it so I figured it's better to use a meta package from the beginning) and have pushed it as well. It will probably take a while before it gets approved as I've noticed that Chocolatey moderation has a big pending queue.
